azerothcore/azerothcore-wotlk
# fix(Core/BWL): Blackwing Lair improvements

## Changes Proposed:
-  Fix Cast method for Nefarian's Tail Whip.
-  Keep Drakonid corpses so they can properly spawn again in 3rd phase.
-  Improved the Drakonid spawner script, moved it to cpp since I needed to.
-  Populate Suppression Room with more whelps, near 120 were missing.
-  Spawn the correct mobs for the trash pack after Vael.
-  Add missing movement for some npcs.
-  Set respawn time of 30m for Krixix.
- Fixed Razorgore immunity mask.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/11124
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/11181
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/11208
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/11178
- Closes https://github.com/chromiecraft/chromiecraft/issues/3280
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/11257

## SOURCE:


## Tests Performed:

- Tested all in-game

## How to Test the Changes:


1. Check Nefarius on wipe/killing adds. All should be the same except for the despawn time for killed adds, they should persist until forever :P Also, with 2 chars, go to Nefarian and check its Tail Whip.
2. Check the trash pack after Vael, each pack should contain 1 Captain, 2 Seether, 1 Flamescale and 2 Wyrmkins.
3. Suppression room should be populated with a lot of whelps.
4. .go c 84557/84558/85756/85759/84840/84625 . They should be moving around.
